src/index.ts

```diff
@@ -116,7 +119,7 @@ export = (app: Application) => {
         await handlePullEvents(context, Container.get(ChallengePullService));
       });
 
-      app.on("schedule.repository", async (context: Context) => {
+      app.on(SCHEDULE_REPOSITORY_EVENT, async (context: Context) => {
```

Comment:
If we only use it once, I think it's OK to use the string.

Reply:
The webhook name in Probot 11 must be in the type definition. So in order to pass the tsc detection, we need to use `any` to define this constant type.

Nevertheless, the warning log still will be output at runtime.

Reply:
Got it.